images/kindnetd/go.mod

```diff
@@ -1,66 +1,77 @@
 module sigs.k8s.io/kind/images/kindnetd
 
-go 1.22.0
+go 1.24.0
+
+toolchain go1.24.2
```

Comment:
Can we manually eliminate this?
IIRC adding this is a go bug that will be fixed.

Reply:
We risk skew here versus .go-version

Reply:
(at least for now, maybe later we will decide to go in on defining in the modules instead, but that's a messy change)